<launch>
  <include file="$(find dogsim)/launch/base_no_sensors.launch">
        <arg name="insitu" value="false"/>
  </include>
  
  <include file="$(find dogsim)/launch/simulated_sensors.launch" />
  <include file="$(find dogsim)/launch/dog.launch" />
  
  <node pkg="dogsim" type="robot_driver" name="robot_driver" output="screen"/>
  <node pkg="dogsim" type="control_dog_position_behavior" name="control_dog_position_behavior" />
  <node pkg="dogsim" type="high_arm_position_action" name="adjust_dog_position_action">
    <param name="torso_height" value="0.06" />
  </node>
</launch>
